### DISCUSSION TITLE
Redirection layout

### DISCUSSION THREAD

**Discussion by akibrhast** [hackforla/website#1272] [Q&A]
What is the purpose of the redirection layout?
@cnk @ExperimentsInHonesty @jbubar @drubgrubby @daniellex0

**Comment by akibrhast** [comment]
Seems to me that it is a legacy existence form when the site was on netlify. I think it could be now removed.
https://github.com/hackforla/website/issues/106

**Comment by jbubar** [comment]
Does this relate to #1308

**Reply by akibrhast** [reply]
yes !
